dotcms-postman/src/main/resources/postman/Category.postman_collection.json

```diff
@@ -1970,11 +1961,754 @@
 									"        }",
 									"    }",
 									"",
-									"pm.expect(isDescending).to.be.true;",
+									"pm.expect(isDescending).to.be.true;",
+									"});",
+									"}"
+								],
+								"type": "text/javascript"
+							}
+						}
+					],
+					"request": {
+						"auth": {
+							"type": "basic",
+							"basic": [
+								{
+									"key": "password",
+									"value": "admin",
+									"type": "string"
```

Comment:
let's use JWT instead of basic auth

Reply:
done https://github.com/dotCMS/core/pull/29091/commits/670ad287e3b850e92fe8add1fb8ddcd4f89d6984